runtime/firecrackeroci/vm.go

```diff
@@ -47,11 +47,11 @@ var (
 	}
 )
 
-// WithVmLocalImageConfig configures a spec with the content of the image's config.
+// WithVMLocalImageConfig configures a spec with the content of the image's config.
 // It is similar to containerd's oci.WithImageConfig except that it does not access
 // the image's rootfs on the host. Instead, it configures what it can on the host and
 // passes information to the agent running in the VM to inspect the image's rootfs.
-func WithVmLocalImageConfig(image containerd.Image) oci.SpecOpts {
+func WithVMLocalImageConfig(image containerd.Image) oci.SpecOpts {
```

Comment:
Need eyes here. Exported function name change is not ideal for a chore commit. Should we exclude the 'revive' linter which is flagging this function name? I was unable to gleam from their documentation how to exclude this instance with an inline comment or similar.

Reply:
I think this is fine. This is a new function with a narrow use-case. I don't think there will be any impact to renaming it.